homeassistant/components/onewire/sensor.py

```diff
@@ -300,18 +302,46 @@ def update(self):
 class OneWireDirect(OneWire):
     """Implementation of a 1-Wire sensor directly connected to RPI GPIO."""
 
+    def _get_valid_data(self):
```

Comment:
We don't accept changes to the parts of this integration that doesn't use a 3rd party library for the device interface.

Reply:
Even for a bug fix?

I was trying to help the community but my own onewire installation is based on the OneWireServer implementation, not the SysBus implementation.

I can migrate it to a random 3rd party library (possibly https://pypi.org/project/pi1wire/) but not sure how to test it on a "real" environment.

Reply:
Yes. All PRs need to be tested before they can be merged.